dockerize.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:latest
+
+ARG DOCKERIZE_VERSION=v0.6.1
```

Comment:
Do we need this? I think it's preferable to just switch over to the "official" image

Reply:
@craig-rueda I don't understand what you mean by "official". I found only this repository on dockerhub (on the attached image). The youngest image from there is 4 years old. If I did not find a link to some official image, then I would be grateful if you show it to me, and I, in turn, could study this issue.

About the busybox image: dockereze requires openssl to work, and busybox  doesn't have a package manager to install it, so in my opinion the best way is to use the alpine base image.
![image](https://github.com/apache/superset/assets/86011874/b6a204ef-b5ae-42b8-bf2e-36a121ad52af)

Reply:
We need this image in order to expect connection to postgreSQL and Redis. If you do not use this image, then users may make an error when starting the chart and will endlessly receive a crashloopbackoff, and this image in the work logs clearly shows that the connection failed and repeats attempts to contact the destination (in my opinion, this is a good idea ).